server/server.js

```diff
@@ -8,7 +8,7 @@ import { CustomError, error, warn } from './utils/utils.js';
 ///Initialization
 // Initialize config
 const __dirname = dirname(fileURLToPath(import.meta.url));
-const _config = JSON.parse(await fs.readFile(path.join(__dirname, './server.config.json')));
+const _config = JSON.parse(await fs.readFile(path.join(__dirname, './server.config.example.json')));
```

Comment:
This should still be "server.config.json" so that "server.config.example.json" can remain for others to refer to (without having real database credentials in it).

Copy-paste "server.config.example.json" and rename it to "server.config.json", and put real database credentials inside of "server.config.json"